Crypto Currencies Complete

luccamicale opened the pull request:
## Compete a simple website Crypto Currencies 💯 

##https://github.com/luccamicale/crypto-currencies 👍 

## Checklist before requesting a review 🔢 
- [x] I have performed a self-review of my code
- [x] use react and redux
- [x] use styles with CSS
- [x] use vercel to do the deploy 🥇 
- [x] use Linters and wepack

![page2](https://user-images.githubusercontent.com/107897361/205100382-387cb86b-21ff-43d9-b002-2ac9f55626fb.PNG)
![page1](https://user-images.githubusercontent.com/107897361/205100392-3fbd9243-2355-4326-9345-691de782daa0.PNG)

luccamicale committed:
branch

luccamicale committed:
add the Navbar

luccamicale committed:
add files

luccamicale committed:
add Navbar and NavLink

luccamicale committed:
more changes

luccamicale committed:
work the navbar

luccamicale committed:
finish navbar with styles

luccamicale committed:
fix linters

luccamicale committed:
created connection with API

luccamicale committed:
more changes

luccamicale committed:
more changes

luccamicale committed:
complete the api

luccamicale committed:
add button back

luccamicale committed:
fix linters

luccamicale committed:
finish details

luccamicale committed:
complete the home

luccamicale committed:
work the input search

luccamicale committed:
more changes

luccamicale committed:
more changess

luccamicale committed:
add button

luccamicale committed:
add readme

luccamicale committed:
more changes

luccamicale committed:
complete the input search

luccamicale committed:
Merge pull request #1 from luccamicale/structure

Structure

luccamicale committed:
Create site

luccamicale committed:
Delete site

luccamicale committed:
Create _redirects

luccamicale committed:
Update _redirects

luccamicale committed:
Delete _redirects

luccamicale committed:
created structure

luccamicale committed:
fix problem

luccamicale committed:
complete the test

luccamicale committed:
Merge pull request #2 from luccamicale/testing

created structure Test

luccamicale committed:
initial commit

luccamicale committed:
add license

luccamicale committed:
fix license

luccamicale committed:
add readme photo

luccamicale committed:
more changes

luccamicale committed:
fix linters

luccamicale committed:
more changes

luccamicale committed:
Merge pull request #3 from luccamicale/styles

Styles and Details

luccamicale committed:
fix netlify

luccamicale committed:
continue problem

luccamicale committed:
fix it

luccamicale committed:
fix it

luccamicale committed:
FIXX IT

luccamicale committed:
add words

luccamicale committed:
fix linters

luccamicale committed:
fix readme and link

prolajumokeoni commented on README.md:
```diff
@@ -236,50 +99,22 @@ Contributions, issues, and feature requests are welcome!
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
```
- [ ]  Based on the requirements your  "Acknowledgements" section or similar should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), please add it 👍🏿

prolajumokeoni commented on src/styles/home.css:
```diff
@@ -0,0 +1,68 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 100px;
+}
```
- [ ] It would be better to make your application look like the design on the mobile screen size, please pay attention to screen sizes from `375px` above your mobile screen looks good from `410px`, please focus more on the mobile screen sizes this will make your design look more like the Behance design and look great on mobile devices, see video below 👍🏿 



https://user-images.githubusercontent.com/69638013/205116997-b495e4b7-db61-409e-a669-3aec5a145244.mp4

prolajumokeoni reviewed (changes requested):
Hi @luccamicale 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
 
✔️  Good Readme




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

luccamicale committed:
modified mobile version and Acknowledgement

luccamicale commented:
I fixed the problem. I modified the styles in the mobile version and I add the name of creator. 🥇 
Now my project should be approved!!! 💯 

![mobile](https://user-images.githubusercontent.com/107897361/205125122-c8319da6-2b2e-472b-8a2e-07ed3fcd3280.PNG)

julie-ify reviewed (approved):
Hi @luccamicale,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
